aztec/src/main/kotlin/org/wordpress/aztec/AztecParser.kt

```diff
@@ -368,7 +369,19 @@ class AztecParser(val plugins: List<IAztecPlugin> = ArrayList()) {
     }
 
     private fun withinNestable(out: StringBuilder, text: Spanned, start: Int, end: Int,
-                               nestable: IAztecSpan, parents: ArrayList<IAztecNestable>?, nestingLevel: Int) {
+                               nestable: IAztecParagraphStyle, parents: ArrayList<IAztecNestable>?, nestingLevel: Int) {
+
+        CssStyleFormatter.removeStyleAttribute(nestable.attributes, CssStyleFormatter.CSS_TEXT_ALIGN_ATTRIBUTE)
```

Comment:
It's not clear to me why we're removing the text-align style attribute, and then re-applying it in case `nestable.shouldParseAlignmentToHtml()` is true.

Reply:
Hmm, I'm trying to think of a case when this could be a problem. It's not an issue for captions because the plugin removes the style attribute. But I suppose we can move it within the if statement.